### ISSUE TITLE
Gurobi::LinExpr::addTerms

### ISSUE THREAD

**Issue by ctriley** [ubnt-intrepid/rust-gurobi#47]
Currently, it appears that in order to build complex linear expressions terms must be added one at a time. This is probably not efficient from a runtime perspective. Gurobi provides an add_terms() method which takes coefficients and variables and allows you to build a linear expression more than one term at a time. Can we expose this functionality in the rust api?

**Comment by Darksonn** [comment]
The [`gurobi::LinExpr`][1] type does not interact directly with gurobi, and is more of a collection type. Its `add_term` method is merely a push to a vector. You can use `set_coeffs` to update many coefficients at once, although you should be aware that #46 has not yet been published.

[1]: https://docs.rs/gurobi/0.3/gurobi/struct.LinExpr.html

**Comment by ctriley** [comment]
still wouldn't something like this, be more efficient?

extending a vector is going to be way better than pushing each term on individually.

```  /// Add multiple linear terms into the expression.
  pub fn add_terms(mut self, coeffs: Vec<f64>, vars: Vec<Var>) -> Self {
    self.coeff.extend(coeff);
    self.vars.extend(var);
    self
  }```

**Comment by Darksonn** [comment]
It wont have any real impact on your performance, but if you think it would be more ergonomic to provide a method that takes slices, that's fine with me.

**Comment by Darksonn** [comment]
Closed by #49.
